streams/src/test/java/org/apache/kafka/streams/integration/RocksDBMetricsIntegrationTest.java

```diff
@@ -181,40 +181,6 @@ public void shouldExposeRocksDBMetricsForSegmentedStateStoreBeforeAndAfterFailur
         );
     }
 
-    @Test
```

Comment:
This is just for my own understanding: for line 144 / 172 above, what kind of simulated failure did we introduce? Seems like we just start and close the stream app while verifying metrics twice?

Reply:
This tests mainly verifies this bug https://issues.apache.org/jira/browse/KAFKA-9355 but also generally whether the RocksDB metrics are exposed. The fix for the bug is hard to verify as a unit test because it involves a couple of components, e.g., `Metered*Store`, `*Segments`, and `RocksDBStore`.

Reply:
Got it, thanks!